src/kartikeyDJune20/PalindromeString.java

```diff
@@ -0,0 +1,21 @@
+package kartikeyDJune20;
+
+public class PalindromeString {
+
+	static void checkStringPalindrome(String input) {
+		String rev = "";
+		for (int index = input.length() - 1; index >= 0; index--) {
+			rev = rev + String.valueOf(input.charAt(index));
```

Comment:
Here, we don't need of String.valurOf() method as we concat the string first and then char, but if it's reverse then we will go this method. No impact on logic .

Reply:
yes yes.. Thanku Harsh.. it should be optimized.

Reply:
Changes Incorporated..